packages/fiori/src/ViewSettingsDialog.js

```diff
@@ -111,7 +111,7 @@ const metadata = {
 	},
 	slots: /** @lends  sap.ui.webcomponents.fiori.ViewSettingsDialog.prototype */ {
 		/**
-		 * Defines the <code>sortItems</code> list.
+		 * Defines the sort items against which the user could sort data.
```

Comment:
I suggest:
* Defines the items against which the user could sort data.
or
* Defines the list of items against which the user could sort data.